what4/src/What4/Protocol/Online.hs

```diff
@@ -169,6 +183,11 @@ data SolverProcess scope solver = SolverProcess
     -- ^ The amount of time (in seconds) that a solver should spend
     -- trying to satisfy any particular goal before giving up.  A
     -- value of zero indicates no time limit.
+
```

Comment:
Should we just remove this?

Reply:
I updated the comment: the original message was from my analysis stage and I forgot to update it based on subsequent activity.  This field is still in use for generic solver code, but it should be set from the solver-specific configuration, and some code will still continue to use the latter.  Let me know if the updated comment is not clear.